src/components/Projects/Echo.tsx

```diff
@@ -0,0 +1,28 @@
+import { AccentText, BodyText } from 'components/Text'
+import EchoLogo from 'icons/EchoLogo'
+import ProjectBlock from 'components/Projects/ProjectBlock'
+
+export default function () {
+  return (
+    <ProjectBlock
+      logo={<EchoLogo />}
+      titleColor="text-secondary"
+      titleText="Echo"
+      subtitle="Tweet anonymously using your ZK badges"
+      description={
+        <BodyText>
+          <AccentText color="text-accent">SealCred</AccentText>{' '}
+          <AccentText color="text-secondary">| Echo</AccentText> allows you to
+          spill the tea using a pseudonymous identity built from your ZK badges.
+          Imagine creating a ZK badge of your work email and tweeting 100%
+          anonymously O_O
+        </BodyText>
+      }
+      buttonUrl="https://echo.sealcred.xyz/"
+      buttonTitle="Create an Echo"
+      imageSource="img/echo-badge.svg"
+      customGlareBorderRadius="5% 5% 46% 46%"
```

Comment:
Looks strange. What does it mean, and what means 5 or 46%?

Reply:
Take a look at the card border radiuses 
![image](https://user-images.githubusercontent.com/49658988/192354122-aa56c6ac-d64b-4338-bf02-e4efc6f0d361.png)
actually this is a custom path, but we can use 5% on top and 46% on the bottom to achieve similar form for glare effect (if we don't pass this prop, the glare will be square)

Reply:
Maybe it's better to use boolean variable instead, like `circle` or `fullRounded` because we have only two types of borders for badges